wp-content/themes/goonj-crm/functions.php

```diff
@@ -372,11 +372,35 @@ function goonj_handle_user_identification_form() {
 }
 
 function goonj_is_volunteer_inducted( $volunteer ) {
+	$optionValue = \Civi\Api4\OptionValue::get(TRUE)
+	->addWhere('option_group_id:name', '=', 'activity_type')
+	->addWhere('label', '=', 'Induction')
+	->execute();
+
+	$activityTypeId = $optionValue->first()['value'];
+
+	$completeStatusOptionValue = \Civi\Api4\OptionValue::get(TRUE)
+	->addWhere('option_group_id:name', '=', 'activity_status')
+	->addWhere('label', '=', 'completed')
+	->setLimit(25)
+	->execute();
+
+	$completeStatusId = $completeStatusOptionValue->first()['value'];
+
+	$unknownStatusOptionValue = \Civi\Api4\OptionValue::get(TRUE)
+	->addWhere('option_group_id:name', '=', 'activity_status')
+	->addWhere('label', '=', 'Unknown')
+	->setLimit(25)
+	->execute();
+
+	$unknownStatusId = $unknownStatusOptionValue->first()['value'];
+
 	$activityResult = \Civi\Api4\Activity::get(FALSE)
 	->addSelect('id')
 	->addWhere('target_contact_id', '=', $volunteer['id'])
-	->addWhere('activity_type_id', '=', 57) // hardcode ID of activity type "Induction"
-	->addWhere('status_id', '=', 2) // hardcode ID of activity status "Completed"
+	->addWhere('activity_type_id', '=', $activityTypeId)
+	->addWhere('status_id', 'CONTAINS', '')
+	->addClause('OR', ['status_id', '=', $completeStatusId], ['status_id', '=', $unknownStatusId])
```

Comment:
Can't we simplify this to:
```suggestion
	->addWhere('status_id:label', 'IN', ['completed', 'Unknown'])
```